atsign-foundation/at_app_cli
# ci: Match directory structure to at_mono

**- What I did**

- Moved all composite actions from `.github/workflows` to `.github/composite`.
- Renamed all references to these jobs in CI.yaml

This is a style change to match at_mono.
Benefit of this style change is that is makes it easy to distinguish a "library" (composite) action from ordinary workflows.

**- How I did it**

Local changes

**- How to verify it**
Name change reflected in the checks:

![CleanShot 2022-10-11 at 14 13 50](https://user-images.githubusercontent.com/33691921/195168230-dc40ee59-a94f-4424-a565-ec2bf646b03f.png)

See that the CI still works as expected on this PR.

**- Description for the changelog**
ci: Match directory structure to at_mono
